app/src/main/java/com/example/easysurvey/dao/ISurveyDao.kt

```diff
@@ -0,0 +1,11 @@
+package com.example.easysurvey.dao
+
+import com.example.easysurvey.dto.Survey
+import retrofit2.Call
+import retrofit2.http.GET
+
+interface ISurveyDao {
+
+    @GET("example/website")
```

Comment:
Added example portion of website into @GET(). This is typically where the rest of the website name after the domain name would go.

Reply:
This looks like an incomplete feature.
Are you using these anywhere?  I don't see them used.
If not in use, remove them.  Unused code is clutter, and bad documentation, which increases technical debt.
Only complete features, which meet the team's Definition of Done, should be merged to Main/Master.